test/unittest/layers/unitest_layer_node.cpp

```diff
@@ -0,0 +1,94 @@
+// SPDX-License-Identifier: Apache-2.0
+/**
+ * Copyright (C) 2020 Parichay Kapoor <pk.kapoor@samsung.com>
+ *
+ * @file        unittest_nntrainer_layers.cpp
```

Comment:
Could you check the file name? s/unitest/unittest/

Reply:
done.